https://bugs.gentoo.org/689530

libva commit 93c52a17b8d2 ("va_backend: remove unneeded
linux/videodev2.h include") removed an include that provided the
prototype of v4l2_format among other v4l2_* structs.

Without this or a forward declaration of 'struct v4l2_format', the prototype
and definition of vdpau_CreateSurfaceFromV4L2Buf() (which has a 'struct
v4l2_format *' parameter) are seen as conflicting.

Just include the removed header where we need it.

diff --git a/src/vdpau_driver.h b/src/vdpau_driver.h
index 6286d16..54d37fb 100644
--- a/src/vdpau_driver.h
+++ b/src/vdpau_driver.h
@@ -21,6 +21,8 @@
 #ifndef VDPAU_DRIVER_H
 #define VDPAU_DRIVER_H
 
+#include <linux/videodev2.h>
+
 #include <va/va_backend.h>
 #include "vaapi_compat.h"
 #include "vdpau_gate.h"
